adagrams/game.py

```diff
@@ -1,13 +1,100 @@
 from random import randint
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+SCORE_CHART = {
+1 : ['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T' ],
+2 : ['D', 'G'],
+3 : ['B', 'C', 'M', 'P'],
+4 : ['F', 'H', 'V', 'W', 'Y'],
+5 : ['K'],	
+8 : ['J', 'X'],
+10 : ['Q', 'Z']
+}
 
 def draw_letters():
-    pass
+    # letters = (LETTER_POOL.keys())
+    # freq_of_letters = (LETTER_POOL.values())
+    pool = []
+    for letter , freq in LETTER_POOL.items():
+        for freq_count in range(freq):
+            pool.append(letter)
+    hand = []
+    for draw_count in range(10):
+        index_val = randint(0,len(pool)-1)
+        hand.append(pool[index_val])
+        pool.pop(index_val)
+    return hand
 
 def uses_available_letters(word, letter_bank):
-    pass
+    word = word.upper()
+    letter_bank_copy = letter_bank.copy()
+    for char in word:
```

Comment:
We can directly call `upper` on `word` when we loop:
```suggestion
    letter_bank_copy = letter_bank.copy()
    for char in word.upper():
```
Why we'd prefer to call upper on line 58:
-It conveys intent to other developers because it's clear that you're working with the uppercase version of the word throughout the loop.